test/parallel/test-async-wrap-GH13045.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+const common = require('../common');
+
+// Checks for the issue in Refs.
```

Comment:
Sorry if my previous comment was unclear. I just meant to write this as:

Refs: https://github.com/nodejs/node/issues/13045